Sources_v3/StreamChatUI/Common Views/AvatarView.swift

```diff
@@ -47,7 +54,14 @@ open class AvatarView: UIView {
     }
     
     open func setupLayout() {
+        imageView.setContentCompressionResistancePriority(.defaultLow, for: .horizontal)
+        imageView.setContentCompressionResistancePriority(.defaultLow, for: .vertical)
         embed(imageView)
-        widthAnchor.constraint(equalTo: heightAnchor, multiplier: 1).isActive = true
+    }
+    
+    // MARK: - AppearanceSetting
+
+    public static func initialAppearanceSetup(_ view: AvatarView) {
+        view.defaultIntrinsicContentSize = .init(width: 40, height: 40)
     }
```

Comment:
What do you think about starting a habit of putting these into a standalone extension?

Reply:
put to extension in c74de15cc35e6f3778c97729e52b844870e4813b, af50f91a41cb2cb784f4d76a0f5802a520564c56, and ffc43d6fd9e14ca0d6984b00311c2ce5e4c851f3